libs/core-ui/src/lib/Highchart/bubbleChart/buildScatterTemplate.ts

```diff
@@ -44,3 +48,21 @@ export function buildScatterTemplate(
   customData.Index = index;
   return customData;
 }
+
+function getAxisValueMapping(
+  axisProperty: string,
+  axisValue: number,
+  axisMap?: { [key: number]: string }
+): string | number {
+  if (axisMap && axisProperty === JointDataset.ClassificationError) {
```

Comment:
I thought this should apply to all categorical values not just classification error.

Reply:
Thanks for catching this @gaugup! Updated to add categorical features.